services/static-webserver/client/source/class/osparc/MaintenanceTracker.js

```diff
@@ -95,6 +95,12 @@ qx.Class.define("osparc.MaintenanceTracker", {
     },
 
     __setMaintenance: function(maintenanceData) {
+      // ignore old maintenance
+      if (new Date(maintenanceData.end).getTime() < new Date().getTime()) {
+        console.error(`Old maintenance "${maintenanceData.reason}" wasn't removed"`);
```

Comment:
since you are handling it and has no side effects, shouldn't be a warning?

Reply:
it's a console message...